lambda/permissionUtil.js

```diff
@@ -16,7 +16,19 @@ exports.decodeJWT = async function (event) {
     decoded = await new Promise(function (resolve) {
       // TODO: This was temporarily necessary for getting UT's to pass.
       if (process.env.IS_OFFLINE == true) {
-        resolve({ resource_access: { 'attendance-and-revenue': { roles: ['sysadmin'] } } });
+        if (event.headers?.PsuedoToken) {
+          if (event.headers.PsuedoToken === "error") {
+            throw new Error("Bad token");
+          } else {
+            resolve(event.headers.PsuedoToken);
+          }
+        } else {
+          resolve({
+            resource_access: {
+              "attendance-and-revenue": { roles: ["sysadmin"] },
+            },
+          });
+        }
```

Comment:
I would suggest putting chunks of this code block into a separate section instead of blurring the lines between what is considered `IS_OFFLINE` and when we're in CI/CD testing mode.  e.g. `if (process.env.IS_TESTING === true)` to always look for the `PsuedoToken` in the header.

My reasoning is that it should be clear if we're running tests `(yarn test)` vs. a developer running the environment `(yarn start)` on their local machine during normal development.

Reply:
I've added `IS_TEST` to the startup script for testing and checking for `process.env.IS_TEST` to make the separation you've brought up. However, I've kept the checking for `PsuedoToken` as it may not be clear to future test writers that when testing the `PsuedoToken` will overwrite the default header.